tests/core/lib/test_smtp.py

```
@@ -0,0 +1,48 @@
+    def close(self):
+        return None
+
+
+class TestSmtpMethod(TestCase):
+    @patch("nettacker.core.lib.smtp.SmtpLibrary.client")
+    def test_brute_force_password(self, mock_smtp_client):
+        library = SmtpLibrary()
+        HOST = "mail.example.com"
+        PORT = SMTP_SESSION_PORT
+        USERNAME = "admin"
+        PASSWORD = "Password@123"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "PASSWORD"

(S105)